packages/vow/test/test-disconnect.js

```diff
@@ -52,18 +52,18 @@ test('retry on disconnection', async t => {
     [2, 'disco', 'disco', 'sad'],
   ];
 
-  for await (const watchVow of [false, true]) {
-    t.log('testing watchVow', watchVow);
+  for await (const doWatch of [false, true]) {
+    t.log('testing watch', doWatch);
     for await (const [final, ...plan] of PLANS) {
-      t.log(`testing (plan=${plan}, watchVow=${watchVow})`);
+      t.log(`testing (plan=${plan}, watch=${doWatch})`);
 
       /** @type {import('../src/types.js').Vow<string>} */
       const vow = makeTagged('Vow', {
         vowV0: makeTestVowV0(plan),
       });
 
       let resultP;
-      if (watchVow) {
+      if (doWatch) {
         const resultW = watch(vow, {
```

Comment:
What is the purpose of the `t.is('then' in resultW, false, 'watch resultW.then is undefined')` a dozen lines below this? Is it important that `resultW` is _not_ thenable, or that it _is_ a vow?